src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -176,13 +176,14 @@ const NetworkModificationNodeEditor = () => {
 
     const cleanClipboard = useCallback(() => {
         setCopyInfos(null);
-        setCopiedModifications((old) => {
-            if (old.length > 0) {
+        setCopiedModifications((oldCopiedModifications) => {
+            if (oldCopiedModifications.length > 0) {
```

Comment:
you can simplify to 
`if (oldCopiedModifications.length) {`

Reply:
fixed